### PR TITLE
Improve Tor documentation

### PR DESCRIPTION
**Pull request by n0toose**

### CODE REVIEW

**Comment by n0toose** [comment]
TO-DO: Note how redirects also work when using an HTTPS certificate.

**Review by helix2301 — APPROVED** [review]
Just cleaning up the Tor documentation rewording things and fixing typos

**Comment by vmstan** [comment]
@n0toose there are some good changes here, are you able to rebase so we can review?

**Comment by n0toose** [comment]
> there are some good changes here, are you able to rebase so we can review?

Thanks for getting back to me. Just so you know: Your attentiveness and efforts with making this project more community-friendly have not gone unnoticed, thanks!

I tried rebasing. Keep in mind: It's been a while. I am not able to test the instructions or 100% back up the veracity of its contents, but, if I remember correctly, the changes were not that technical.

File: content/en/admin/optional/tor.md [hunk 1]
```diff
@@ -126,17 +128,17 @@ map $http_upgrade $connection_upgrade {
...
   include /etc/nginx/snippets/mastodon.conf;
 
-  ssl_certificate /etc/letsencrypt/live/mastodon.myhosting.com/fullchain.pem;
-  ssl_certificate_key /etc/letsencrypt/live/mastodon.myhosting.com/privkey.pem;
+  ssl_certificate /etc/letsencrypt/live/mastodon.example.com/fullchain.pem;
+  ssl_certificate_key /etc/letsencrypt/live/mastodon.example.com/privkey.pem;
 }
 ```
 
 Replace the long hash provided here with your Tor domain located in the file at `/var/lib/tor/onion_service/hostname`.
```
Comment (vmstan) [inline content/en/admin/optional/tor.md:139]:
```suggestion
Replace the long hash provided here with your Tor domain located in the file at `/var/lib/tor/hidden_service/hostname`.
```

**Review by vmstan — CHANGES_REQUESTED** [review]

File: content/en/admin/optional/tor.md (hunk 1 above)
Comment (n0toose) [reply content/en/admin/optional/tor.md]: Will need to do a cleanup + rebase over the next few days.

**Review by vmstan — COMMENTED** [review]
Ignore this.

**Review by andypiper — APPROVED** [review]

**Pull request merged**